lib/apiservers/engine/backends/container.go

```diff
@@ -890,7 +845,7 @@ func (c *Container) cleanupPortBindings(vc *viccontainer.VicContainer) error {
 			}
 
 			log.Debugf("Unmapping ports for powered off / removed container %q", mappedCtr)
-			err = UnmapPorts(cc.ContainerID, vc)
+			err = networking.UnmapPorts(cc.ContainerID, vc)
```

Comment:
I would consider using name "network", not networking. It is shorted and has an explicit meaning.